CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 7.0.0 - unreleased
+
+- remove `poolboy` dependency
+- improve ETS backend efficiency ~15x
```

Comment:
Under load it's actually x40+

x15 (according to Benchee) is when Poolboy is not overloaded